stdlib/file/src/main/ballerina/file/service_endpoint.bal

```diff
@@ -37,7 +37,7 @@ public type Listener object {
 
     public extern function start();
 
-    public extern function stop();
+    //public extern function stop();
```

Comment:
What is the reason for commenting out this?

Reply:
Endpoint life cycle not working at the moment. So this stop method won't call in current implementation.